### ISSUE TITLE
Can you update some examples？

### ISSUE THREAD

**Issue by Ko-Hon** [espressif/esp-matter#329]
- may update some example like :

sensors: temperature / humanity / light

conrtrol :Thermostat/ fan 

thanks。

**Comment by dhrishi** [comment]
@Ko-Hon The esp-matter data model APIs should be straightforward to add any device types. Is there a specific thing you are looking into or facing issues?

**Comment by Ko-Hon** [comment]
yes but i don't know how, So I want to develop, or learn, based on a few examples. This makes it faster to get started

see：https://github.com/project-chip/connectedhomeip/tree/master/examples

**Comment by lboue** [comment]
> * may update some example like :
> 
> sensors: temperature / humanity / light

You mean humidity?

**Comment by dhrishi** [comment]
@Ko-Hon The examples in this repo should be simpler to add as you can dynamically create the data model using the APIs. Please see this as a reference https://github.com/espressif/esp-matter/issues/361#issuecomment-1530915395
